src/app/components/EditorInterface.tsx

```diff
@@ -204,8 +207,26 @@ const EditorInterface = () => {
     appState?.setEditorColor(
       (appState?.appState.settings as ISettings).backgroundColor,
     )
+    appState?.setBannerImgUrl(
+      (appState?.appState.settings as ISettings).bannerImage.url,
+    )
   }, [appState?.appState.settings])
 
+  useEffect(() => {
+    ;(async () => {
```

Comment:
`;`

Reply:
seems like prettier automatically puts this